contracts/interfaces/IEnum.sol

```diff
@@ -3,9 +3,9 @@ pragma solidity >=0.7.0 <0.9.0;
 
 /**
  * @title Enum - Collection of enums used in Safe contracts.
- * @author Richard Meissner - @rmeissner
+ * @author Shebin John - @remedcu
```

Comment:
I'd double check what's our internal policy on authorship...

Reply:
Authors should not be removed (even for copied code). But additional authors can be added.

But to be honest, we should probably change this to just state that the author is the Safe Core team or similar.

Reply:
```
* author Safe Protocol - @safe-global/safe-protocol
```

Wdyt?

Reply:
can we just have the github handle? `* author @safe-global/safe-protocol`

Reply:
Absolutely 👍🏾 Should I make the change for all the contracts of just the contracts being changed in this PR?

Reply:
Based on internal discussion, only changing the contracts author with changes in this PR.